htdocs/libraries/icms/ipf/view/Table.php

```diff
@@ -656,20 +676,20 @@ public function render($fetchOnly = false, $debug = false) {
 
 		$this->setSortOrder();
 
-		$this->_limitsel = !empty($limitsel) ? $limitsel : icms_getCookieVar($script_name . '_limitsel', '15');
+		$this->_limitsel = $_GET['limitsel'] ?? $this->getCookie('limitsel', '15');
```

Comment:
?? works only in PHP 7.0

Reply:
changed it to isset($_GET['limitsel']) ? $_GET['limitsel'] :  $this->getCookie('limitsel', '15');